src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements Area, Draw {
+    private String color;
+    private int side1;
+    private int side2;
```

Comment:
```suggestion
    private int firstSide;
    private int secondSide;
```
please avoid numbers in variable names